aspnet/fundamentals/logging.rst

```diff
@@ -73,23 +96,37 @@ Exception error
 
 .. note:: Some loggers, such as the built-in ``ConsoleLogger`` used in this article, will ignore the ``eventId`` parameter. If you need to display it, you can include it in the message string. This is done in the following sample so you can easily see the eventId associated with each message, but in practice you would not typically include it in the log message.
```

Comment:
We probably shouldn't tell people to do this. Production logging frameworks will have options for including the eventId if you want it. ConsoleLogger is intended to be human readable.